src/components/LocationToCompare.js

```diff
@@ -55,12 +59,20 @@ const LocationCompare = () => {
         );
       })}
       </div>
-      <ComparisonButton onClick={() => {}} />
-      <div>
-        <p style={disclaimerText}>Add up to 4 regions.</p>
-      </div>
+      {showButton && (
+        <>
+          <ComparisonButton onClick={() => { history.push('/comparison') }} />
```

Comment:
I'd rather we use a react-router-dom <Link to="/comparison"> here instead of history.push(), as we've been using Link everywhere else so far.

The onClick prop of ComparisonButton can be made optional.